main.go

```diff
@@ -68,7 +72,7 @@ func main() {
 	}
 
 	if configTokens != nil && *configTokens != "" {
-		cfg, err := ConfigTokens(*configTokens, *configPath)
+		cfg, err := cfg.ConfigTokens(*configTokens, *configPath)
```

Comment:
This doesn't cause an issue have a local var called `cfg` and a package called `cfg`?

Reply:
it's just a local shadowing, have not noticed, but I changed it